src/types/markers.js

```diff
@@ -144,6 +155,17 @@ export type MarkerSchema = {|
 
   // if present, give the marker its own local track
   graphs?: Array<MarkerGraph>,
+
+  // If set to true, markers of this type are assumed to be well-nested with all
+  // other stack-based markers on the same thread. Stack-based markers may
+  // be displayed in a different part of the marker chart than non-stack-based
+  // markers.
+  // Instant markers are always well-nested.
```

Comment:
Instant markers are already displayed on a separate line anyway.

Reply:
Well, this is proposing that we put stack-based instant markers into the pyramid.